packages/orbit-components/src/Separator/index.tsx

```diff
@@ -80,18 +80,24 @@ const Separator = ({
   spaceAfter,
   type = "solid",
   color,
+  label,
 }: Props) => {
   return (
-    <div className={cx("box-border w-full", getSideOffsetAmount(sideOffset, align))}>
+    <div className={cx("relative min-h-px", spaceAfter && getSpaceAfterClasses(spaceAfter))}>
       <hr
         className={cx(
           "orbit-separator",
-          "mt-0 box-border h-0 border-t",
+          "absolute left-0 top-1/2 mt-0 box-border -translate-y-1/2 border-t",
           color || "border-elevation-flat-border-color",
+          getSideOffsetAmount(sideOffset, align),
           BORDER_TYPE_CLASSES[type],
-          spaceAfter && getSpaceAfterClasses(spaceAfter),
         )}
       />
+      {label && (
+        <span className="bg-white-normal px-100 -translate-x-50 absolute start-1/2 top-1/2 min-w-max -translate-y-1/2">
```

Comment:
Label text inside the separator is not centered:
<img width="625" alt="image" src="https://github.com/user-attachments/assets/1c3270c0-0a45-4ea3-8c67-55bc75aaa689">

Reply:
You should use the Text element, not direct string

Reply:
In Storybook it won't be possible, but that's for display purposes only. The prop is documented as accepting ReactNode, so devs should be aware of the correct usage

Reply:
ReactNode also includes `string` prop, so it's possible to add a string there as well. Also, in the [documentation](https://dsil-add-label-separator.orbit.pages.dev/components/structure/separator/react/), there is `string` type for `label` prop mentioned.

Although the text is wrapped by the `<Text>` component, it's not still centered.

Reply:
Yes, documentation needs a change. And yes, a string is possible, but that does not mean we have to enforce the styles. On a Design System, one is expected to render text using typography components, no direct strings, especially if it's documented.

I see the Text centered with the `<Text>` component
<img width="660" alt="image" src="https://github.com/user-attachments/assets/671ff56b-c4b6-4b6e-8a6b-213dd7f8654a">

Reply:
If you mean vertical-align, then it's correct, I mean horizontal-align, which is still incorrect. :( 
<img width="1120" alt="image" src="https://github.com/user-attachments/assets/49e50655-abf4-423d-b946-809d87c469e9">

Reply:
Ah, got it, it was a wrong class. Fixed. Visual tests updating

Reply:
Fixed